server/asset/common.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+package asset
+
+type Error string
+
+func (err Error) Error() string { return string(err) }
+
+const (
+	UnsupportedScriptError = Error("unsupported script type")
+)
+
+// The DEXAsset interface is an interface for a blockchain backend. The DEX
+// primarily needs to track UTXOs and transactions to validate orders, and
+// to monitor community conduct.
+type DEXAsset interface {
+	// UTXO should return a UTXO only for outputs that would be spendable on the
+	// blockchain immediately. Pay-to-script-hash UTXOs require the redeem script
```

Comment:
What is meant by "spendable on the blockchain immediately".  A wallet can spend an unconfirmed UTXO, so it is not clear what qualifies as spendable in this context.